manifests/container.pp

```diff
@@ -39,7 +42,7 @@
   $https_content = epp('foreman_proxy_content/container-gateway-fragment.epp', $context)
 
   apache::vhost::fragment { 'pulp-https-container':
-    vhost    => 'pulpcore-https',
+    vhost    => $pulpcore_https_vhost,
```

Comment:
I wonder why this isn't using [pulpcore::apache::fragment](https://github.com/theforeman/puppet-pulpcore/blob/master/manifests/apache/fragment.pp) instead. Then you don't need to specify the vhost. Otherwise it should at least reuse `$pulpcore::apache::http_vhost_name`.

Reply:
It's called with

```puppet
  if $enable_docker {
    include pulpcore::plugin::container
    unless $shared_with_foreman_vhost {
      class { 'foreman_proxy_content::container':
        pulpcore_https_vhost => $apache_https_vhost,
      }

      class { 'foreman_proxy::plugin::container_gateway':
        pulp_endpoint => "https://${servername}",
      }
    }
  }
```

Where `$apache_https_vhost` is the same that is passed to the `pulpcore` class:

```puppet
  class { 'pulpcore':
    ...
    apache_https_vhost             => $apache_https_vhost,
    ...
  }
```

I would be happy to see the `foreman_proxy_content::container` class changed to use `pulpcore::apache::fragment` but perhaps in a second PR as it isn't strictly necessary for this feature.